charts/argo-tunnel/values.yaml

```diff
@@ -11,7 +11,7 @@ controller:
   resources:
     limits:
       cpu: 100m
-      memory: 128Mi
+      memory: 256M
```

Comment:
I had worried about that (especially without any specific memory footprint testing on this version). Can you change to Mi to get the power 2 variant? And then I’ll merge.

Reply:
Woops i meant Mi